docs/Generics/chapters/types.tex

```diff
@@ -759,7 +759,7 @@ \section{Source Code Reference}\label{typesourceref}
 \item \texttt{ArchetypeType}, and its three subclasses:
 \begin{itemize}
 \item \texttt{PrimaryArchetypeType},
-\item \texttt{OpenedArchetypeType},
+\item \texttt{ExistentialArchetypeType},
```

Comment:
You didn’t have to update this, but thanks! There’s a whole unfinished chapter on existential types I’ll get around to finishing some day…